test/v2/config/environment/testnet.json

```diff
@@ -0,0 +1,18 @@
+{
+  "SRSPath": "~/.test-v2/srs",
+  "KeyPath": "~/.test-v2/testnet-private-key.txt",
+  "DisperserHostname": "disperser-testnet-holesky.eigenda.xyz",
+  "DisperserPort": 443,
+  "EthRPCURLs": [
+    "https://ethereum-holesky-rpc.publicnode.com"
+  ],
+  "BLSOperatorStateRetrieverAddr":          "0x5322F480c6a14aBA6491974fa0fbF2A925C71B94",
+  "EigenDAServiceManagerAddr":              "0xD4A7E1Bd8015057293f0D0A557088c286942e84b",
+  "EigenDACertVerifierAddressQuorums0_1":   "0xFe52fE1940858DCb6e12153E2104aD0fDFbE1162",
+  "EigenDACertVerifierAddressQuorums0_1_2": "",
+  "EigenDACertVerifierAddressQuorums2":     "",
+  "SubgraphURL": "https://subgraph.satsuma-prod.com/51caed8fa9cb/eigenlabs/eigenda-operator-state-preprod-holesky/version/v0.7.0/api",
```

Comment:
Looks like the preprod graph url. The testnet one should be similar but with preprod removed I think.

Reply:
fixed